Makefile

```diff
@@ -7,14 +7,13 @@ build: build_dashboard
 r_build:
 	docker build --no-cache --pull -t forecast-eval-build docker_build
 
-predictions_cards.rds score_cards_state_deaths.rds score_cards_state_cases.rds score_cards_nation_cases.rds score_cards_nation_deaths.rds: dist
-	test -f dist/$@ || curl -o dist/$@ $(S3_URL)/$@ 
+%.rds: dist
+	test -f dist/$@ || curl -o dist/$@ $(S3_URL)/$@
 
-pull_data: predictions_cards.rds score_cards_state_deaths.rds score_cards_state_cases.rds score_cards_nation_cases.rds score_cards_nation_deaths.rds
+pull_data: score_cards_state_deaths.rds score_cards_state_cases.rds score_cards_nation_cases.rds score_cards_nation_deaths.rds score_cards_state_hospitalizations.rds score_cards_nation_hospitalizations.rds
```

Comment:
are these part of the styling changes or did something else sneak in?

Reply:
when I was locally testing the setup, it didn't pull all the local data. Thus I changed this line to have all the fallback data which was defined in the `getAllData`

Reply:
This seems fine. Although we upload `predictions_cards.rds` to the S3 bucket, it isn't used for anything in the dashboard.